lib/mix/tasks/still.compile.ex

```diff
@@ -5,9 +5,17 @@ defmodule Mix.Tasks.Still.Compile do
   def run(_) do
     {:ok, _} = Application.ensure_all_started(:still)
 
+    Still.Compiler.CompilationQueue.subscribe()
+
     Application.put_env(:still, :url_fingerprinting, true)
     Application.put_env(:still, :dev_layout, false)
 
     Still.Compiler.Traverse.run()
+
+    receive do
+      :bus_empty -> :ok
+    after
+      :infinity -> :error
+    end
```

Comment:
Is this intended?

Reply:
Yap, otherwise the compilation fails after 5 seconds.